### ISSUE TITLE
Unable to connect to the dependency-check database

### ISSUE THREAD

**Issue by mgladdish** [albuch/sbt-dependency-check#81]
**Describe the bug**

I'd previously been running dependencyCheck quite happily on my Mac. But now I've switched to running a Fedora VM under QubesOS it no longer seems to work.

I get the error

```
[error] (*:dependencyCheck) org.owasp.dependencycheck.exception.ExceptionCollection: One or more exceptions occurred during analysis:
[error]         Unable to connect to the dependency-check database
```

**Version of sbt-dependency-check used**

The problem occurs using version 0.2.10 (which is what I was using on my mac) and 1.3.0 of the the plugin.

**Log file**

https://gist.github.com/mgladdish/3971948c0fd101d9a1ab67fd7f4d6307

**To Reproduce**

1. execute the `dependencyCheck` target

**Expected behavior**

The dependencyCheck task to run without errors, and produce a report.

**Additional context**

I've seen similar errors logged on the "parent" project, e.g. https://github.com/jeremylong/DependencyCheck/issues/1783. I tried `dependencyCheckPurge` and deleting my `.ivy2/cache/org.owasp/dependency-check-utils/jars/data` directory. Neither of which helped.

**Comment by albuch** [comment]
Hi @mgladdish, could you provide the portion of your build.sbt regarding configuration of sbt-dependency-check along with the output of `dependencyCheckListSettings`?

**Comment by mgladdish** [comment]
Hi @albuch,

Of course.

Our `build.sbt` only sets the following:

```
dependencyCheckSuppressionFile := Some(new File("DependencyCheckSuppressions.xml"))
```

and `dependencyCheckListSettings` produces

```
[accounts] $ dependencyCheckListSettings
[pool-14-thread-1] DEBUG org.owasp.dependencycheck.utils.Settings - Properties loaded:

analyzer.cocoapods.enabled='true'
proxy.disableSchemas='true'
cve.url.modified.validfordays='7'
cve.url.base='https://nvd.nist.gov/feeds/json/cve/1.0/nvdcve-1.0-%d.json.gz'
analyzer.dependencybundling.enabled='true'
cve.url.original='https://nvd.nist.gov/feeds/json/cve/1.0/nvdcve-1.0-modified.json.gz'
data.connection_string='jdbc:h2:file:%s;AUTOCOMMIT=ON;LOG=0;CACHE_SIZE=65536;'
archive.scan.depth='3'
analyzer.cmake.enabled='true'
odc.application.name='Dependency-Check Core'
analyzer.retirejs.repo.validforhours='24'
analyzer.node.package.enabled='true'
analyzer.central.url='https://search.maven.org/solrsearch/select'
downloader.quick.query.timestamp='true'
analyzer.retirejs.repo.js.url='https://raw.githubusercontent.com/Retirejs/retire.js/master/repository/jsrepository.json'
analyzer.archive.enabled='true'
cpe.url='https://static.nvd.nist.gov/feeds/xml/cpe/dictionary/official-cpe-dictionary_v2.3.xml.gz'
analyzer.ossindex.url='https://ossindex.sonatype.org'
analyzer.nvdcve.enabled='true'
analyzer.jar.enabled='true'
analyzer.cpe.enabled='true'
cve.url-1.2.modified='https://nvd.nist.gov/feeds/xml/cve/1.2/nvdcve-modified.xml.gz'
analyzer.central.query='%s?q=1:%s&wt=xml'
cve.url-1.2.base='https://nvd.nist.gov/feeds/xml/cve/1.2/nvdcve-%d.xml.gz'
analyzer.bundle.audit.enabled='true'
analyzer.node.audit.use.cache='true'
analyzer.dependencymerging.enabled='true'
data.h2.shutdownhook='org.owasp.dependencycheck.utils.H2DBCleanupHook'
cpe.validfordays='30'
junit.fail.on.cvss='0'
analyzer.node.audit.url='https://registry.npmjs.org/-/npm/v1/security/audits'
odc.application.version='5.2.1'
analyzer.msbuildproject.enabled='true'
data.user='dcuser'
engine.version.url='https://jeremylong.github.io/DependencyCheck/current.txt'
analyzer.autoconf.enabled='true'
odc.autoupdate='true'
cve.url-2.0.base='https://nvd.nist.gov/feeds/xml/cve/2.0/nvdcve-2.0-%d.xml.gz'
analyzer.cpesuppression.enabled='true'
analyzer.retirejs.enabled='true'
cve.check.validforhours='4'
cve.url-2.0.modified='https://nvd.nist.gov/feeds/xml/cve/2.0/nvdcve-2.0-Modified.xml.gz'
analyzer.versionfilter.enabled='true'
analyzer.assembly.enabled='true'
cve.cpe.startswith.filter='cpe:2.3:a:'
analyzer.golang.dep.enabled='true'
odc.analysis.timeout='30'
analyzer.nexus.url='https://repository.sonatype.org/service/local/'
cve.url-2.0.original='https://nvd.nist.gov/feeds/xml/cve/2.0/nvdcve-2.0-Modified.xml.gz'
analyzer.hint.enabled='true'
analyzer.retired.enabled='false'
database.batchinsert.enabled='true'
analyzer.openssl.enabled='true'
analyzer.swift.package.manager.enabled='true'
analyzer.nuspec.enabled='true'
analyzer.central.retry.count='7'
analyzer.central.use.cache='true'
updater.nvdcve.enabled='true'
max.download.threads='2'
analyzer.central.enabled='true'
analyzer.python.distribution.enabled='true'
analyzer.nexus.enabled='false'
odc.reports.pretty.print='false'
analyzer.artifactory.enabled='false'
analyzer.retirejs.filternonvulnerable='false'
analyzer.ossindex.enabled='true'
analyzer.central.parallel.analysis='false'
analyzer.nexus.proxy='true'
cve.startyear='2002'
downloader.tls.protocols='TLSv1.1,TLSv1.2,TLSv1.3'
data.file_name='odc.mv.db'
analyzer.ruby.gemspec.enabled='true'
analyzer.composer.lock.enabled='true'
data.version='4.1'
analyzer.vulnerabilitysuppression.enabled='true'
data.directory='[JAR]/data'
analyzer.experimental.enabled='false'
analyzer.golang.mod.enabled='true'
analyzer.filename.enabled='true'
analyzer.ossindex.use.cache='true'
analyzer.python.package.enabled='true'
data.driver_path=''
updater.versioncheck.enabled='true'
ecosystem.skip.cpeanalyzer='npm'
analyzer.falsepositive.enabled='true'
analyzer.node.audit.enabled='true'
data.driver_name='org.h2.Driver'
database.batchinsert.maxsize='1000'
data.password='*****'
analyzer.nugetconf.enabled='true'
cve.url.modified='https://nvd.nist.gov/feeds/json/cve/1.0/nvdcve-1.0-modified.json.gz'

[info] Applying project settings to DependencyCheck settings
[pool-14-thread-1] DEBUG org.owasp.dependencycheck.utils.Settings - Setting: odc.application.name='accounts'
[pool-14-thread-1] DEBUG org.owasp.dependencycheck.utils.Settings - Setting: suppression.file='["/home/user/code/uxforms-accounts-api/DependencyCheckSuppressions.xml"]'
[pool-14-thread-1] DEBUG org.owasp.dependencycheck.utils.Settings - Setting: analyzer.central.enabled='false'
[info] Running list-settings for accounts
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer Archive Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer File Name Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer Jar Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer Hint Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer CPE Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer False Positive Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer Dependency Bundling Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer Dependency Merging Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer NVD CVE Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer Vulnerability Suppression Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer Central Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer Nexus Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer Artifactory Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer Nuspec Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer Nugetconf Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer MSBuild Project Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer Assembly Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer OpenSSL Source Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer Node.js Package Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer Node Audit Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer RetireJS Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer Ruby Bundle Audit Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer Version Filter Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.analyzer.AnalyzerService - Loaded Analyzer Sonatype OSS Index Analyzer
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.utils.Settings - Settings.getDataFile() - file: '[JAR]/data'
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.utils.Settings - Settings.getDataFile() - transforming filename
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.utils.Settings - Settings.getDataFile() - jar file: '/home/user/.ivy2/cache/org.owasp/dependency-check-utils/jars'
[pool-14-thread-2] DEBUG org.owasp.dependencycheck.utils.Settings - Settings.getDataFile() - returning: '/home/user/.ivy2/cache/org.owasp/dependency-check-utils/jars/data'
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCacheManager getUnconfiguredInstance
INFO: Instance is null, returning unconfigured instance
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.utils.threadpool.ThreadPoolManager loadConfig
INFO: thread_pool.default PoolConfiguration = useBoundary = [false] boundarySize = [2000] maximumPoolSize = [150] minimumPoolSize = [4] keepAliveTime = [300000] whenBlockedPolicy = [RUN] startUpSize = [4]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCacheManager doConfigure
INFO: Setting default auxiliaries to ODC
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCacheManager doConfigure
INFO: setting defaultCompositeCacheAttributes to [ useLateral = true, useRemote = true, useDisk = true, maxObjs = 0, maxSpoolPerRun = -1, diskUsagePattern = UPDATE, spoolChunkSize = 2 ]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCacheManager doConfigure
INFO: setting defaultElementAttributes to [ IS_LATERAL = false, IS_SPOOL = true, IS_REMOTE = false, IS_ETERNAL = false, MaxLifeSeconds = 86400, IdleTime = 1800, CreateTime = 1565629498712, LastAccessTime = 1565629498712, getTimeToLiveSeconds() = 86399, createTime = 1565629498712 ]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.memory.AbstractDoubleLinkedListMemoryCache initialize
INFO: initialized MemoryCache for POM
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCache <init>
INFO: Constructed cache with name [POM] and cache attributes [ useLateral = true, useRemote = true, useDisk = true, maxObjs = 0, maxSpoolPerRun = -1, diskUsagePattern = UPDATE, spoolChunkSize = 2 ]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.AuxiliaryCacheConfigurator parseCacheEventLogger
INFO: No cache event logger defined for auxiliary [jcs.auxiliary.ODC]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.AuxiliaryCacheConfigurator parseElementSerializer
INFO: Using standard serializer [org.apache.commons.jcs.utils.serialization.StandardSerializer@313c6c0a] for auxiliary [jcs.auxiliary.ODC]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.utils.threadpool.ThreadPoolManager loadConfig
INFO: thread_pool.disk_cache_event_queue PoolConfiguration = useBoundary = [false] boundarySize = [2000] maximumPoolSize = [150] minimumPoolSize = [4] keepAliveTime = [300000] whenBlockedPolicy = [RUN] startUpSize = [4]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.disk.indexed.IndexedDiskCache initializeFileSystem
INFO: Region [POM] Cache file root directory: /home/user/.ivy2/cache/org.owasp/dependency-check-utils/jars/data/cache
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.disk.indexed.IndexedDiskCache initializeKeyMap
INFO: Region [POM] Set maxKeySize to: '1000000'
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.disk.indexed.IndexedDiskCache <init>
INFO: Region [POM] Indexed Disk Cache is alive.
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.memory.AbstractDoubleLinkedListMemoryCache initialize
INFO: initialized MemoryCache for NODEAUDIT
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCache <init>
INFO: Constructed cache with name [NODEAUDIT] and cache attributes [ useLateral = true, useRemote = true, useDisk = true, maxObjs = 0, maxSpoolPerRun = -1, diskUsagePattern = UPDATE, spoolChunkSize = 2 ]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.AuxiliaryCacheConfigurator parseCacheEventLogger
INFO: No cache event logger defined for auxiliary [jcs.auxiliary.ODC]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.AuxiliaryCacheConfigurator parseElementSerializer
INFO: Using standard serializer [org.apache.commons.jcs.utils.serialization.StandardSerializer@78664a53] for auxiliary [jcs.auxiliary.ODC]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.disk.indexed.IndexedDiskCache initializeFileSystem
INFO: Region [NODEAUDIT] Cache file root directory: /home/user/.ivy2/cache/org.owasp/dependency-check-utils/jars/data/cache
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.disk.indexed.IndexedDiskCache initializeKeyMap
INFO: Region [NODEAUDIT] Set maxKeySize to: '1000000'
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.disk.indexed.IndexedDiskCache <init>
INFO: Region [NODEAUDIT] Indexed Disk Cache is alive.
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.memory.AbstractDoubleLinkedListMemoryCache initialize
INFO: initialized MemoryCache for CENTRAL
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCache <init>
INFO: Constructed cache with name [CENTRAL] and cache attributes [ useLateral = true, useRemote = true, useDisk = true, maxObjs = 0, maxSpoolPerRun = -1, diskUsagePattern = UPDATE, spoolChunkSize = 2 ]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.AuxiliaryCacheConfigurator parseCacheEventLogger
INFO: No cache event logger defined for auxiliary [jcs.auxiliary.ODC]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.AuxiliaryCacheConfigurator parseElementSerializer
INFO: Using standard serializer [org.apache.commons.jcs.utils.serialization.StandardSerializer@49e6f8c3] for auxiliary [jcs.auxiliary.ODC]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.disk.indexed.IndexedDiskCache initializeFileSystem
INFO: Region [CENTRAL] Cache file root directory: /home/user/.ivy2/cache/org.owasp/dependency-check-utils/jars/data/cache
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.disk.indexed.IndexedDiskCache initializeKeyMap
INFO: Region [CENTRAL] Set maxKeySize to: '1000000'
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.disk.indexed.IndexedDiskCache <init>
INFO: Region [CENTRAL] Indexed Disk Cache is alive.
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCacheConfigurator parseRegions
INFO: Parsed regions [POM, NODEAUDIT, CENTRAL]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCacheManager doConfigure
INFO: Finished configuration in 40 ms.
[info] 	dependencyCheckAutoUpdate: true
[info] 	dependencyCheckCveValidForHours: 4
[info] 	dependencyCheckFailBuildOnCVSS: 11.0
[info] 	dependencyCheckJUnitFailBuildOnCVSS: 0.0
[info] 	dependencyCheckFormats (combined with dependencyCheckFormat): HTML
[info] 	dependencyCheckOutputDirectory: /home/user/code/uxforms-accounts-api/target/scala-2.11
[info] 	dependencyCheckScanSet: /home/user/code/uxforms-accounts-api/src/main/resources
[info] 	dependencyCheckSkip: false
[info] 	dependencyCheckSkipTestScope: true
[info] 	dependencyCheckSkipRuntimeScope: false
[info] 	dependencyCheckSkipProvidedScope: false
[info] 	dependencyCheckSkipOptionalScope: false
[info] 	dependencyCheckSuppressionFile/s: ["/home/user/code/uxforms-accounts-api/DependencyCheckSuppressions.xml"]
[info] 	dependencyCheckHintsFile: 
[info] 	dependencyCheckAnalysisTimeout: 30
[info] 	dependencyCheckEnableExperimental: false
[info] 	dependencyCheckEnableRetired: false
[info] 	dependencyCheckArchiveAnalyzerEnabled: true
[info] 	dependencyCheckZipExtensions: null
[info] 	dependencyCheckJarAnalyzer: true
[info] 	dependencyCheckCentralAnalyzerEnabled: false
[info] 	dependencyCheckCentralAnalyzerUseCache: true
[info] 	dependencyCheckOSSIndexAnalyzerEnabled: true
[info] 	dependencyCheckOSSIndexAnalyzerUrl: https://ossindex.sonatype.org
[info] 	dependencyCheckOSSIndexAnalyzerUseCache: true
[info] 	dependencyCheckOSSIndexAnalyzerUsername: null
[info] 	dependencyCheckOSSIndexAnalyzerPassword: ******
[info] 	dependencyCheckNexusAnalyzerEnabled: false
[info] 	dependencyCheckNexusUrl: https://repository.sonatype.org/service/local/
[info] 	dependencyCheckNexusUsesProxy: true
[info] 	dependencyCheckNexusUser: null
[info] 	dependencyCheckNexusPassword: ******
[info] 	dependencyCheckPyDistributionAnalyzerEnabled: true
[info] 	dependencyCheckPyPackageAnalyzerEnabled: true
[info] 	dependencyCheckRubygemsAnalyzerEnabled: true
[info] 	dependencyCheckOpensslAnalyzerEnabled: true
[info] 	dependencyCheckCmakeAnalyzerEnabled: true
[info] 	dependencyCheckAutoconfAnalyzerEnabled: true
[info] 	dependencyCheckComposerAnalyzerEnabled: true
[info] 	dependencyCheckNodeAnalyzerEnabled: true
[info] 	dependencyCheckNodeAuditAnalyzerEnabled: true
[info] 	dependencyCheckNodeAuditAnalyzerUrl: https://registry.npmjs.org/-/npm/v1/security/audits
[info] 	dependencyCheckNodeAuditAnalyzerUseCache: true
[info] 	dependencyCheckNuspecAnalyzerEnabled: true
[info] 	dependencyCheckNugetConfAnalyzerEnabled: true
[info] 	dependencyCheckCocoapodsEnabled: true
[info] 	dependencyCheckSwiftEnabled: true
[info] 	dependencyCheckBundleAuditEnabled: true
[info] 	dependencyCheckPathToBundleAudit: 
[info] 	dependencyCheckBundleAuditWorkingDirectory: null
[info] 	dependencyCheckAssemblyAnalyzerEnabled: true
[info] 	dependencyCheckPathToDotNETCore: 
[info] 	dependencyCheckCpeStartsWith: cpe:2.3:a:
[info] 	dependencyCheckRetireJSAnalyzerEnabled: true
[info] 	dependencyCheckRetireJSAnalyzerRepoJSUrl: https://raw.githubusercontent.com/Retirejs/retire.js/master/repository/jsrepository.json
[info] 	dependencyCheckRetireJsAnalyzerRepoValidFor: 24
[info] 	dependencyCheckRetireJsAnalyzerFilters: null
[info] 	dependencyCheckRetireJsAnalyzerFilterNonVulnerable: false
[info] 	dependencyCheckArtifactoryAnalyzerEnabled: false
[info] 	dependencyCheckArtifactoryAnalyzerUrl: null
[info] 	dependencyCheckArtifactoryAnalyzerUseProxy: false
[info] 	dependencyCheckArtifactoryAnalyzerParallelAnalysis: false
[info] 	dependencyCheckArtifactoryAnalyzerUsername: null
[info] 	dependencyCheckArtifactoryAnalyzerApiToken: null
[info] 	dependencyCheckArtifactoryAnalyzerBearerToken: null
[info] 	dependencyCheckGolangDepEnabled: true
[info] 	dependencyCheckGolangModEnabled: true
[info] 	dependencyCheckPathToGo: 
[info] 	dependencyCheckCveUrlModified: https://nvd.nist.gov/feeds/json/cve/1.0/nvdcve-1.0-modified.json.gz
[info] 	dependencyCheckCveUrlBase: https://nvd.nist.gov/feeds/json/cve/1.0/nvdcve-1.0-%d.json.gz
[info] 	dependencyCheckConnectionTimeout: null
[info] 	dependencyCheckDatabaseFileName: odc.mv.db
[info] 	dependencyCheckDatabaseVersion: 4.1
[info] 	dependencyCheckDataDirectory: [JAR]/data
[info] 	dependencyCheckDatabaseDriverName: org.h2.Driver
[info] 	dependencyCheckDatabaseDriverPath: 
[info] 	dependencyCheckConnectionString: jdbc:h2:file:%s;AUTOCOMMIT=ON;LOG=0;CACHE_SIZE=65536;
[info] 	dependencyCheckDatabaseUser: dcuser
[info] 	dependencyCheckDatabasePassword: ******
[info] 	dependencyCheckUseSbtModuleIdAsGav: false
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.event.ElementEventQueue dispose
INFO: Element event queue destroyed: org.apache.commons.jcs.engine.control.event.ElementEventQueue@2c9ba124
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCache dispose
INFO: In DISPOSE, [NODEAUDIT] fromRemote [false]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCache dispose
INFO: In DISPOSE, [NODEAUDIT] auxiliary [NODEAUDIT]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCache dispose
INFO: In DISPOSE, [NODEAUDIT] put 0 into auxiliary NODEAUDIT
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.disk.AbstractDiskCache run
INFO: No longer waiting for event queue to finish: Pooled Cache Event Queue
Working = true
Alive = false
Empty = true
Queue Size = 0
Queue Capacity = 2147483647
Pool Size = 0
Maximum Pool Size = 150
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.disk.AbstractDiskCache dispose
INFO: In dispose, destroying event queue.
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.disk.indexed.IndexedDiskCache saveKeys
INFO: Region [NODEAUDIT] Saving keys to: NODEAUDIT, key count: 0
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.disk.indexed.IndexedDiskCache saveKeys
INFO: Region [NODEAUDIT] Finished saving keys.
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.disk.indexed.IndexedDiskCache disposeInternal
INFO: Region [NODEAUDIT] Shutdown complete.
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCache dispose
INFO: In DISPOSE, [NODEAUDIT] disposing of memory cache.
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.memory.AbstractMemoryCache dispose
INFO: Memory Cache dispose called.
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCache dispose
INFO: In DISPOSE, [CENTRAL] fromRemote [false]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCache dispose
INFO: In DISPOSE, [CENTRAL] auxiliary [CENTRAL]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCache dispose
INFO: In DISPOSE, [CENTRAL] put 0 into auxiliary CENTRAL
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.disk.AbstractDiskCache run
INFO: No longer waiting for event queue to finish: Pooled Cache Event Queue
Working = true
Alive = false
Empty = true
Queue Size = 0
Queue Capacity = 2147483647
Pool Size = 0
Maximum Pool Size = 150
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.disk.AbstractDiskCache dispose
INFO: In dispose, destroying event queue.
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.disk.indexed.IndexedDiskCache saveKeys
INFO: Region [CENTRAL] Saving keys to: CENTRAL, key count: 0
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.disk.indexed.IndexedDiskCache saveKeys
INFO: Region [CENTRAL] Finished saving keys.
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.auxiliary.disk.indexed.IndexedDiskCache disposeInternal
INFO: Region [CENTRAL] Shutdown complete.
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCache dispose
INFO: In DISPOSE, [CENTRAL] disposing of memory cache.
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.memory.AbstractMemoryCache dispose
INFO: Memory Cache dispose called.
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCache dispose
INFO: In DISPOSE, [POM] fromRemote [false]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCache dispose
INFO: In DISPOSE, [POM] auxiliary [POM]
Aug 12, 2019 6:04:58 PM org.apache.commons.jcs.engine.control.CompositeCache dispose
INFO: In DISPOSE, [POM] put 0 into auxiliary POM
Aug 12, 2019 6:04:59 PM org.apache.commons.jcs.auxiliary.disk.AbstractDiskCache run
INFO: No longer waiting for event queue to finish: Pooled Cache Event Queue
Working = true
Alive = false
Empty = true
Queue Size = 0
Queue Capacity = 2147483647
Pool Size = 0
Maximum Pool Size = 150
Aug 12, 2019 6:04:59 PM org.apache.commons.jcs.auxiliary.disk.AbstractDiskCache dispose
INFO: In dispose, destroying event queue.
Aug 12, 2019 6:04:59 PM org.apache.commons.jcs.auxiliary.disk.indexed.IndexedDiskCache saveKeys
INFO: Region [POM] Saving keys to: POM, key count: 0
Aug 12, 2019 6:04:59 PM org.apache.commons.jcs.auxiliary.disk.indexed.IndexedDiskCache saveKeys
INFO: Region [POM] Finished saving keys.
Aug 12, 2019 6:04:59 PM org.apache.commons.jcs.auxiliary.disk.indexed.IndexedDiskCache disposeInternal
INFO: Region [POM] Shutdown complete.
Aug 12, 2019 6:04:59 PM org.apache.commons.jcs.engine.control.CompositeCache dispose
INFO: In DISPOSE, [POM] disposing of memory cache.
Aug 12, 2019 6:04:59 PM org.apache.commons.jcs.engine.memory.AbstractMemoryCache dispose
INFO: Memory Cache dispose called.
[success] Total time: 1 s, completed Aug 12, 2019 6:04:59 PM
```

**Comment by albuch** [comment]
Unfortunately the default for sbt/scala is to only show the exception stacktrace and not showing the stacktraces and messages of the underlying causes. The root cause in this case is wrapped two times in new Exceptions.
I'm preparing a fix to log those exceptions including their root cause.

**Comment by albuch** [comment]
@mgladdish thanks for your patience. I've added logging for this case and I'm going to prepare a new release later today.

**Comment by albuch** [comment]
Just published v1.3.1 which should hit Maven Central shortly.

**Comment by albuch** [comment]
@mgladdish I'm going to close this issue for now. if you need more assistance please leave a comment
